### PR TITLE
Delete user api

### PR DESCRIPTION
**Pull request by kamran-gafoor**

### CODE REVIEW

File: Routes/UserRoute.js [hunk 1]
```diff
@@ -53,4 +53,19 @@ router.route("/adduser").post((req, res) => {
...
 
+//Endpoint to delete a new user
+router.route("/delete/:id").delete((req, res) => {
+  user
+    .findByIdAndDelete(req.params.id)
+    .then((user) => {
+      if (!user) {
+        return res.status(404).json("User not Find with ID:" + user);
+      }
+      res.send(user);
```
Comment (saahilanande) [inline Routes/UserRoute.js]: Send string saying user is Deleted successfully with his id instead of the user object of the deleted user

**Review by saahilanande — CHANGES_REQUESTED** [review]
Check line 64 for changes

**Comment by kamran-gafoor** [comment]
Resolved

**Review by saahilanande — APPROVED** [review]
Looks good

**Pull request merged**